Tests/govuk_ios/govuk_ios_snapshot_tests/Helpers/SnapshotTestCase.swift

```diff
@@ -0,0 +1,121 @@
+import UIKit
+
+import iOSSnapshotTestCase
+
+@testable import govuk_ios
+
+class SnapshotTestCase: FBSnapshotTestCase {
+
+    override func setUp() {
+        super.setUp()
+//        self.recordMode = true
+    }
+
+    func VerifySnapshotInWindow(_ viewController: UIViewController,
+                                perPixelTolerance: CGFloat = 0,
+                                overallTolerance: CGFloat = 0,
```

Comment:
Interesting, I was going to ask if there is a way to get some tolerance on this. But, I totally get that we want pixel-perfect 👍

Reply:
We probably won't want pixel perfect forever, but until it breaks, might as well start there